src/deploy/scope_subscription.ts

```diff
@@ -54,21 +54,24 @@ export async function DeploySubscriptionScope(azPath: string, validationOnly: bo
     // validate the deployment
     core.info("Validating template...")
     var code = await exec(`"${azPath}" deployment sub validate ${azDeployParameters} -o json`, [], validateOptions);
-    if (validationOnly && code != 0) {
+    if (deploymentMode.toLowerCase() === "validate" && code != 0) {
```

Comment:
rather than doing `.toLowerCase()` everywhere can't we just change once when creating `deploymentMode` variable in main.ts.
But check that the az command works with small case value for mode

Reply:
sounds good
I'll verify and update